### PR TITLE
change all cli tools to use read-text-file package for encoding

### PR DESCRIPTION
**Pull request by tomlm**
Fixes: #168 #165 
Change CLI tools to use read-text-file package to correctly handle UTF16 encoded text files

### CODE REVIEW

File: MSBot/bin/msbot-connect-azure.js [hunk 1]
```diff
@@ -98,7 +98,6 @@ async function processConnectAzureArgs(config) {
             endpoint: args.endpoint
         });
         config.connectService(endpointService);
-        process.stdout.write(`Connected ${endpointService.type}:${endpointService.endpoint}\n`);
         services.push(endpointService);
```
Comment (vishwacsena) [inline MSBot/bin/msbot-connect-azure.js]: Why are we removing this? I liked knowing when a service was successfully added to my .bot file..

File: MSBot/bin/msbot-connect-dispatch.js [hunk 2]
```diff
@@ -59,18 +59,20 @@ async function processConnectDispatch(config) {
...
-        Object.assign(args, JSON.parse(fs.readFileSync(args.input, 'utf8')));
+        Object.assign(args, JSON.parse(await txtfile.read(args.input)));
     }
     if (!args.hasOwnProperty('name'))
         throw new Error('Bad or missing --name');
     if (!args.appId || !utils_1.uuidValidate(args.appId))
         throw new Error('bad or missing --appId');
-    if (!args.version || parseInt(args.version) == 0)
+    if (!args.version)
         throw new Error('bad or missing --version');
```
Comment (vishwacsena) [inline MSBot/bin/msbot-connect-dispatch.js]: Good fix to have since the LUIS version is really a string.

File: Ludown/lib/toLU.js [hunk 3]
```diff
@@ -122,7 +122,7 @@ var parseQnAJSONFile = function(file){
         process.stderr.write(chalk.default.redBright('Sorry unable to open [' + file + ']\n'));        
         process.exit(retCode.FILE_OPEN_ERROR);
     }
-    var QNAFileContent = readFile.readSync(file);
+    var QNAFileContent = txtfile.readFileSync(file);
```
Comment (tomlm) [inline Ludown/lib/toLU.js]: >readFileSync [](start = 33, length = 12)

changed to .readSync()

File: Ludown/lib/parser.js [hunk 4]
```diff
@@ -5,14 +5,15 @@
 const fs = require('fs');
 const path = require('path');
 const chalk = require('chalk');
+const txtfile = require('read-text-file');
```
Comment (tomlm) [inline Ludown/lib/parser.js]: I removed duplicate

**Pull request merged**